rc_django/views.py

```diff
@@ -46,24 +47,20 @@ def proxy(request, service, url):
     use_pre = False
     headers = {}
 
+    if service == "iasystem":
+        if url.endswith('/evaluation'):
```

Comment:
Set the header here?
headers["Accept"] = "application/vnd.collection+json"

Reply:
Ah, added it back. Thank you!